connector/rest/src/main/java/it/pagopa/selfcare/dashboard/connector/rest/decoder/FeignErrorDecoder.java

```diff
@@ -0,0 +1,17 @@
+package it.pagopa.selfcare.dashboard.connector.rest.decoder;
+
+import feign.Response;
+import feign.codec.ErrorDecoder;
+import it.pagopa.selfcare.dashboard.connector.exception.ResourceNotFoundException;
+
+public class FeignErrorDecoder extends ErrorDecoder.Default {
+
+    @Override
+    public Exception decode(String methodKey, Response response) {
+        if (response.status() == 404) {
+            throw new ResourceNotFoundException(response.reason());
+        } else {
```

Comment:
you could handled also 500 error like as https://github.com/pagopa/selfcare-onboarding-backend/blob/main/connector/rest/src/main/java/it/pagopa/selfcare/onboarding/connector/rest/decoder/FeignErrorDecoder.java